gcloud-java-compute/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>gcloud-java-compute</artifactId>
+  <packaging>jar</packaging>
+  <name>GCloud Java compute</name>
+  <description>
+    Java idiomatic client for Google Cloud Compute Engine.
+  </description>
+  <parent>
+    <groupId>com.google.gcloud</groupId>
+    <artifactId>gcloud-java-pom</artifactId>
+    <version>0.1.5-SNAPSHOT</version>
+  </parent>
+  <properties>
+    <site.installationModule>gcloud-java-compute</site.installationModule>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>gcloud-java-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.apis</groupId>
+      <artifactId>google-api-services-compute</artifactId>
+      <version>v1-rev93-1.21.0</version>
```

Comment:
From the public docs, it looks like there's a v1 and a beta version out.  Just so that I'm on the same page, why did we choose to go with v1 over beta?

Also it looks like a new revision was released a couple days ago: [v1-rev97-1.21.0](http://search.maven.org/#artifactdetails%7Ccom.google.apis%7Cgoogle-api-services-compute%7Cv1-rev97-1.21.0%7Cjar)